.github/workflows/continous-integration.yml

```diff
@@ -973,7 +973,6 @@ jobs:
 
     # deploy will only be run when there is a tag available
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && github.repository == 'RasaHQ/rasa'
-    needs: [quality, test, wait_for_docs_tests, docker] # only run after all other stages succeeded
```

Comment:
Shouldn't we keep docker?

Reply:
Yeah i will add it back but since docker was already built it'll fail this time around and deploy to pypi will again be blocked.

Reply:
Ah gotcha, ok.